ui/imports/shared/popups/send/views/AmountToSend.qml

```diff
@@ -86,6 +87,9 @@ Control {
     property var formatBalance: balance =>
                                 `${balance.toLocaleString(Qt.locale())} CRYPTO`
 
+    /* Shows separator between top and bottom text */
+    property bool showSeparator: false
```

Comment:
There is already one similar flag `dividerVisible`. Please user similar name and group them close to each-other. And please add switch in SB page.

Reply:
Or it's even the same thing and should be de-duplicated?